### PR TITLE
Add support for Node v13

### LINKED ISSUE #1483: Could not work under nodejs 13
## Issue or Feature

Could not install package.

## Your Environment
* Version of node-canvas (output of `npm list canvas` or `yarn list canvas`):2.6.0
* Environment (e.g. node 4.2.0 on Mac OS X 10.8):node v13.0.1 on Debian 10

### PR DESCRIPTION
**Pull request by csvn**
- [x] Have you updated CHANGELOG.md?

I have no idea if anything else is needed, but I thought I'd create a PR and see if Travis builds without errors. I don't think there were any major changes between v12 and v13, so it's not unlikely it might work.

Fixes #1483

### CODE REVIEW

**Comment by csvn** [comment]
The Travis build and test seemed to pass. I'm not sure how binaries are deployed to https://github.com/node-gfx/node-canvas-prebuilt/releases, but let me know and I can do my best to fix it.

**Comment by chearon** [comment]
@csvn there's some info in the wiki on how it's done but maintainers have to do it. I've added 2.6.0 / node 13 builds for Linux / macOS. As is tradition we have to wait for AppVeyor to add node 13 before we can get Windows builds updated.

I'm not sure if we need the update to the README since no code changes were made, but the updated tests is Ince!

I'll try and get 2.6.1 done this weekend too so we can publish that.

**Comment by DominusVilicus** [comment]
Not working for me

**Comment by DominusVilicus** [comment]
Downgrading to node 12 works. (I am using node v13.1 on windows)

**Comment by DominusVilicus** [comment]
Updating to 2.6.1 does not fix the issue. I am on windows 10

**Comment by DominusVilicus** [comment]
AppVeyor supports node 13

**Comment by chearon** [comment]
Just added

`v2.6.1`: Node 12/13 
`v2.6.0`: Node 12

Not sure why the 12s were missing, but 13 was added to AppVeyor in between now and when I last did the builds

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ==================
 ### Changed
 ### Added
+* Support Node.js v13
```
Comment (zbjornson) [inline CHANGELOG.md]: @csvn could you remove this line please so people don't think that a new version of Canvas is required to use Node.js v13? Then we can merge this :)

**Comment by DominusVilicus** [comment]
So does it work now?

File: CHANGELOG.md (hunk 1 above)
Comment (csvn) [reply CHANGELOG.md]: Sure! Removed any changes to `CHANGELOG.md`.

**Comment by tsheaff** [comment]
It seems like this has been merged, but there's no released version of the node-canvas package since it merged on Jan 8th 2020. Are there plans to release a new version of node-canvas that has this change? This is keeping us from being able to upgrade out node version (and I'm sure we're not the only ones).

cc @csvn @zbjornson

**Pull request merged**